src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.WindowsSdkSupportedTargetPlatforms.props

```diff
@@ -13,14 +13,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 <!-- This file contains a list of the windows target platform versions that are supported by this SDK for .NET. -->
 <Project>
     <ItemGroup>
-        <WindowsSupportedTargetPlatform Include="10.0.19041" />
-        <WindowsSupportedTargetPlatform Include="10.0.18362" />
-        <WindowsSupportedTargetPlatform Include="10.0.17763" />
-        <WindowsSupportedTargetPlatform Include="8.0" />
-        <WindowsSupportedTargetPlatform Include="7.0" />
+        <WindowsSdkSupportedTargetPlatform Include="10.0.19041" />
```

Comment:
I'd add a comment here about how these items are used, including the name of the Target that processes them.